pkg/inttest/db.go

```diff
@@ -31,6 +31,7 @@ func SetupDB(t *testing.T) *gorm.DB {
 	db, err := storage.NewDatabase(logger, config.Postgresql{
 		Host:         container.Host,
 		Port:         container.DefaultPort(),
+		SslMode:      "disable",
```

Comment:
Looks like the `mode` is always `require` except for tests. Is this tested? Making exceptions for tests means our code in tests behaves differently than in prod. If we were able to use the same setup during tests we would not have to add that config at all and be more confident that it works and keeps working.

Reply:
The issue is that Gnomock doesn't support any ssl mode but disable. Do you have any idea about how we could test this?

Reply:
Add it to gnomock or use testcontainers are the options I know of